src/other/PropertyEditor.js

```diff
@@ -74,6 +79,20 @@
         ;
         table
             .each(function (d) {
+                if (d instanceof PropertyEditor === false) {
+                    if (context.overrideTags() && Object.keys(context.overrideTags()).length) {
```

Comment:
It should work how you have it. I just meant the code should be abstracted to its own method.